modules/instrument_manager/templates/menu_instrument_manager.tpl

```diff
@@ -17,7 +17,7 @@
     </div>
 </div>
 {/if}
-{if $feedback|count}
+{if isset($feedback) && $feedback|count}
```

Comment:
You can actually just remove the count part and check for `isset`. It does the same thing. Not sure why I originally put `count()` in here, maybe it used to be an array.